### PR TITLE
Pointer support

### PR DESCRIPTION
**Pull request by pranavbaburaj**
Added pointer support to hascal

```dart
var name = "Pranav Baburaj";
print(ptr<name>);
```

Also added `puts` support
```dart
puts("Hello World");
```

Also added a folder structure for build
 - dlang source is stored in `build` and the executable is stored in `build\dist` folder

Hope you will inform me if you find any problems or please merge the commits

Hopefully Pranav

### CODE REVIEW

File: src/h_parser.py [hunk 1]
```diff
@@ -913,6 +914,17 @@ def call_func(self, p):
     def call_func(self, p):
         return "writeln({0})".format(p.params_call)
 
+    @_("PTR LESS params_call GREATER")
+    def call_func(self, p):
+        values = p.params_call.split(",")[0]
+        return f"&{values}"
+
```
Comment (pranavbaburaj) [inline src/h_parser.py:921]: Pointers

File: src/h_parser.py [hunk 2]
```diff
@@ -913,6 +914,17 @@ def call_func(self, p):
...
+    @_('PUTS LPAREN params_call RPAREN')
+    def call_func(self, p):
+        return "write({0})".format(p.params_call)
+
+
+
```
Comment (pranavbaburaj) [inline src/h_parser.py:927]: Puts

**Comment by hasan-kashi** [comment]
This is a good idea.Please wait a few days. We are being rewritten compiler.

**Comment by hasan-kashi** [comment]
Don't close your pull request

**Comment by pranavbaburaj** [comment]
Ok, I will wait. Thank you

**Comment by hasan-kashi** [comment]
Hi,the compiler rewrited,please use the latest source code.
Also project generator was removed from the original code,because project generator should written in Hascal(for example cargo,a project generator for rust programming language written in rust)

**Comment by pranavbaburaj** [comment]
Sure, I will try changing it

**Comment by pranavbaburaj** [comment]
I have resolved the marge conflicts and also added some build guidelines

Please merge the commits or please let me know what is wrong

Hopefully Pranav

**Comment by hasan-kashi** [comment]
Please match your code with the repository code. Your code is out of date.

**Comment by pranavbaburaj** [comment]
I have merged the new commits

Please merge the pull request

**Comment by hasan-kashi** [comment]
>     dlang source is stored in `build` and the executable is stored in `build\dist` folder
remove it

**Comment by hasan-kashi** [comment]
Your h_lexer.py is out of date.

**Comment by hasan-kashi** [comment]
Your code is too old, I will close your pull request.

**Pull request closed without merging**